partials/footer.hbs

```diff
@@ -2,29 +2,28 @@
     <div class="social">
         {{#if @site.facebook}}
             <a class="social-item social-item-facebook"
-                href="{{facebook_url @site.facebook}}" target="_blank"
-                rel="noopener noreferrer" aria-label="Facebook">
+                href="https://www.facebook.com/advisorysg/" target="_blank"
+                rel="noopener noreferrer" aria-label="Facebook" >
                 <svg class="icon">
                     <use xlink:href="#facebook-box"></use>
                 </svg>
             </a>
         {{/if}}
-        {{#if @site.twitter}}
-            <a class="social-item social-item-twitter"
-                href="{{twitter_url @site.twitter}}" target="_blank"
-                rel="noopener noreferrer" aria-label="Twitter">
-                <svg class="icon">
-                    <use xlink:href="#twitter-box"></use>
-                </svg>
-            </a>
-        {{/if}}
-        <a class="social-item social-item-rss"
-            href="https://feedly.com/i/subscription/feed/{{@site.url}}/rss/"
-            target="_blank" rel="noopener noreferrer" aria-label="RSS">
+        <a class="social-item social-item-instagram" 
+            href="https://www.instagram.com/advisory.sg/"
+            target="_blank" rel="noopener noreferrer" aria-label="Instagram">
+            <svg class="icon" style="margin-left:3px ;margin-top:3px">
```

Comment:
Margin shouldn't be set in this manner, instead the actual SVG icon should be edited accordingly. See what I wrote in the review comment for more details.